### PR TITLE
Refactor AssetsInstallCommandListener

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: src/EventListener/AssetsInstallCommandListener.php [hunk 1]
```diff
@@ -37,8 +37,6 @@ final class AssetsInstallCommandListener
     public const METHOD_ABSOLUTE_SYMLINK = 'absolute symlink';
     public const METHOD_RELATIVE_SYMLINK = 'relative symlink';
 
-    protected static $defaultName = 'assets:install';
```
Comment (VincentLanglet) [inline src/EventListener/AssetsInstallCommandListener.php:40]: This is use to run the command `bin/console assets:install` so it's needed.

File: src/EventListener/AssetsInstallCommandListener.php [hunk 2]
```diff
@@ -75,7 +73,7 @@ public function copySonataCoreBundleAssets(ConsoleTerminateEvent $event)
         $this->execute($event->getInput(), $event->getOutput(), $application);
     }
 
-    protected function execute(InputInterface $input, OutputInterface $output, FrameworkApplication $application): int
+    private function execute(InputInterface $input, OutputInterface $output, FrameworkApplication $application): int
```
Comment (VincentLanglet) [inline src/EventListener/AssetsInstallCommandListener.php:76]: This is a BC break.

And the parent method execute is protected.

File: src/EventListener/AssetsInstallCommandListener.php (hunk 1 above)
Comment (VincentLanglet) [reply src/EventListener/AssetsInstallCommandListener.php:40]: Oh, it's not a command

File: src/EventListener/AssetsInstallCommandListener.php (hunk 2 above)
Comment (VincentLanglet) [reply src/EventListener/AssetsInstallCommandListener.php:76]: Oh it's final

File: src/EventListener/AssetsInstallCommandListener.php (hunk 1 above)
Comment (OskarStark) [reply src/EventListener/AssetsInstallCommandListener.php:40]: I thought the same first

File: src/EventListener/AssetsInstallCommandListener.php (hunk 2 above)
Comment (OskarStark) [reply src/EventListener/AssetsInstallCommandListener.php:76]: I thought the same first 😄

File: src/EventListener/AssetsInstallCommandListener.php (hunk 1 above)
Comment (wbloszyk) [reply src/EventListener/AssetsInstallCommandListener.php:40]: It is extension for command. This line can be deleted. IMO better will be use it in line 71 insted.
```php
if ('assets:install' !== $command->getName() || null !== $coreBundle) {
    return;
}
```

**Comment by wbloszyk** [comment]
Looks good for me.

**Pull request merged**